### ISSUE TITLE
[AutoCompleteTextView] Re-announces the Activity Title after Item Selection

### ISSUE THREAD

**Issue by oddalot** [material-components/material-components-android#1962]
**Description:**

Implement the Exposed Dropdown Menu as explained here, under 'Implementing an exposed dropdown menu' https://material.io/components/text-fields/android#using-text-fields

Turn on talkback, and select an item in the dropdown. Talkback will re-announce the title of the Activity after the dropdown item selection. 

Most likely this is due to the `ListPopupWindow` that the `AutoCompleteTextViewUses` being outside the current view hierarchy.

**Expected behavior:**

Talkback should not announce the title of the Activity when using a dropdown.

**Source code:**

Implement the Exposed Dropdown Menu as explained here, under 'Implementing an exposed dropdown menu' https://material.io/components/text-fields/android#using-text-fields

**Android API version:**

API 29

**Material Library version:**

com.google.android.material:material:1.2.1

**Device:**

Emulated Pixel 3a

**Comment by consp1racy** [comment]
Hi, does the platform `AutoCompleteTextView` behave differently?

**Comment by leticiarossi** [comment]
This seems specific to TalkBack not to the autocompletetextview as it happens with other components as well. Talkback announces the activity name (what's defined in `android:label` of your activity manifest), [this post](https://stackoverflow.com/questions/50623326/talkback-reading-the-appname-for-each-activity) has some workaround suggestions such as calling `setTitle` on your activity.
